pkg/languages/js/js.go

```diff
@@ -177,7 +181,27 @@ func runtimeDirectory() string {
 	return path.Join(config.RuntimeDir, "js")
 }
 
-func generatePackageJson(configurationEntries map[string][]string) ([]byte, error) {
+func getPackageJSON(userSourceDir string, configurationEntries map[string][]string) ([]byte, error) {
+	packageJSON, err := generatePackageJSON(configurationEntries)
+	if err != nil {
+		return nil, err
+	}
+	packageJSONFile := path.Join(userSourceDir, "package.json")
+
+	if fileInfo, err := os.Stat(packageJSONFile); err == nil {
```

Comment:
Look inside [util module](https://github.com/openshift-cloud-functions/kfn/blob/master/pkg/util/fs.go), there are already a bunch of functions that wraps usual file system utilities, trying to cover edge cases. I prefer to use these than directly the `os` methods